src/cls/ZPM/Package.cls

```diff
@@ -286,16 +286,30 @@ ClassMethod GetModuleMeta(url As %String) As %DynamicObject
     Set ht.Port = port
   }
   Set ht.Location = tComponents("path")_"master/module.xml"
-    
   $$$ThrowOnError(ht.Get())
   If (ht.HttpResponse.StatusCode'=200) {
     $$$ThrowStatus($$$ERROR(5001,"Unable to download XML "_ht.Location))
   }
-  $$$ThrowOnError(##class(%XML.XPATH.Document).CreateFromStream(ht.HttpResponse.Data, .xpathdoc))
+  Set tmpFolder = $$$FileTempDir
+  Do ##class(%File).CreateDirectory(tmpFolder)
```

Comment:
```suggestion
```
$$$FileTempDir returns already created folder

Reply:
deleted in 2 places in this file